### PR TITLE
Feature: Changed Database to use make use of asynchronous techniques Issue: #2

### PR DESCRIPTION
**Pull request by Thetvdh**
Implemented all commands that were being used in the existing bot. Removed some commands that will probably never be used. Would like some feedback on the overall edits though.

### CODE REVIEW

File: lib/cmds/feeling_lucky.py [hunk 1]
```diff
@@ -18,11 +18,12 @@ def __init__(self, bot):
     @commands.command(name=CMD)
     async def feeling_lucky(self, ctx):
         rand = random.randint(1, 3)
+        message = None
 
         match rand:
             # Item Event - Add an item to the player's inventory
             case 1:
-                item_name, quantity = self.__give_item(ctx.author.id)
+                item_name, quantity = await self.__give_item(ctx.author.id)
```
Comment (Thetvdh) [inline lib/cmds/feeling_lucky.py]: noticed a bug here myself. Will update commit soon

**Comment by OBlount** [comment]
As we are working with aiosqlite, we also need to update the `requirements.txt`

File: lib/cmds/feeling_lucky.py (hunk 1 above)
Comment (OBlount) [reply lib/cmds/feeling_lucky.py]: I will commit this

File: lib/cmds/inventory.py [hunk 2]
```diff
@@ -16,11 +16,11 @@ def __init__(self, bot):
     # string name = CMD
     @commands.command(name=CMD)
     async def list_inventory(self, ctx):
-        data = OPERATOR.inventory_read(ctx.author.id)
+        data = await inventory_read(ctx.author.id)
         message = f"<@{ctx.author.id}>'s inventory:\n"
 
         for item in data:
-            message = message + str(item[0]) + ": " + str(item[1]) + "\n"
+            message = message + f"{item[0]}: {item[1]}\n"
```
Comment (OBlount) [inline lib/cmds/inventory.py:23]: I like this. Nice one

**Comment by OBlount** [comment]
Think this is ready to merge now. Have a quick look yourself, either pull down the commit or code review on the site.

**Review by OBlount — APPROVED** [review]
Think this is ready to merge now. Have a quick look yourself, either pull down the commit or code review on the site.

**Pull request merged**